### ISSUE TITLE
Move the hostname settings from security to network options

### ISSUE THREAD

**Issue by ikem-krueger** [MichaIng/DietPi#7218]
I was running `dietpi-config` and was looking in the network options for an option to change the hostname.

It turns out that the settings is in the menu point `Security`.

That would be the last place where I would look for that.

Hence I suggest to move that to `Network Options: Adapter` or `Network Options: Misc`.

**Comment by MichaIng** [comment]
Indeed, I find that one weird as well, it has been like this since before I joined the project. I'll just duplicate it to `Network Options: Misc` and leave a prompt when opening from security options for some versions, so users are informed, before removing it from there.
